Task/main.cpp

```diff
@@ -0,0 +1,60 @@
+#include <iostream>
+#include <ctime>
+
+#include "array.h"
+
+void print(Array* arr, size_t size)
+{	
+	for(int i = 0; i < size; i++)
+		std::cout << array_get(arr, i) << " ";
+	std::cout << std::endl;
+}
+
+void arrayCompressor(Array* arr, int a, int b, size_t arrSize)
+{
+	Array* newArray = array_create(arrSize);
```

Comment:
Можно ли обойтись без дополнительного массива?

Reply:
Да, можно. Есть такой вариант:
```suggestion
void arrayCompressor(Array* arr, int a, int b, size_t arrSize)
{
	size_t extraNumbers = 0;
	for (int i = 0; i < arrSize; i++)
	{
		if (array_get(arr, i) >= a && array_get(arr, i) <= b)
		{
			extraNumbers++;
			continue;
		}
		array_set(arr, i - extraNumbers, array_get(arr, i));
	}
	
	for (int i = arrSize - extraNumbers; i < arrSize; i++)
		array_set(arr, i, 0);

}
```